### PR TITLE
Added CircleCI

### PR DESCRIPTION
**Pull request by marcferna**

### CODE REVIEW

**Review by skukx — DISMISSED** [review]
Nicely Done

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  publish_gem:
+    steps:
+      - run:
+          name: Publish gem to Github Registry
+          command: gem push --KEY github --host https://rubygems.pkg.github.com/sendoso rubocop-sendoso-*.gem
+
+  delete_registry_credentials:
+    steps:
+      - run:
+          name: Publish gem to Github Registry
```
Comment (marcferna) [inline .circleci/config.yml]: change naming here! Copy paste 🤦

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+version: 2.1
+
+commands:
+  set_registry_credentials:
+    steps:
+      - run:
+          name: Installing Dependencies
```
Comment (marcferna) [inline .circleci/config.yml]: change naming here! Copy paste 🤦

**Comment by marcferna** [comment]
Sorry team! this is not ready to review yet...it automatically picks the code owners and set them as reviewers

**Comment by marcferna** [comment]
Ready now!

The job it will only run when a tag gets pushed. Then, I'll update the README to explain how to deploy new versions. @skukx you could use this same circleci config for the other gems (warden and jwt) if you want. I created a special access token just for this repo though

**Comment by skukx** [comment]
> Ready now!
> 
> The job it will only run when a tag gets pushed. Then, I'll update the README to explain how to deploy new versions. @skukx you could use this same circleci config for the other gems (warden and jwt) if you want. I created a special access token just for this repo though

This is fantastic I may do something similar to how we release to auth0 for the other gems:
https://github.com/sendoso/auth0-configuration/blob/master/.github/workflows/release.yml

**Review by skukx — APPROVED** [review]

**Pull request merged**